inc/native/namespace.php

```diff
@@ -17,8 +17,16 @@
  * Setup integration.
  */
 function bootstrap() {
+	// Only load native analytics solution for Accelerate or higher.
+	if ( Altis\get_feature_tier() === 'plus' ) {
```

Comment:
Seeing as we don't currently have this data set up, can we use analytics.enabled here instead? Given this is in the analytics module, we can control the tiering at the higher level

Reply:
That's essentially what's required to get to this function anyway.

Reply:
So how can I tell what's happening at a higher level if not via the tier constants?

Reply:
Yeah, so would make sense to just drop the check I think then?